modules/primer-marketing-utilities/lib/margin.scss

```diff
@@ -0,0 +1,80 @@
+// Margin spacer utilities
+// stylelint-disable block-opening-brace-space-before, declaration-colon-space-before, primer/selector-no-utility, comment-empty-line-before
+@for $i from 1 through length($spacers) {
```

Comment:
Likewise here, we don't need this full loop since it will generate CSS for allll the spacer variables instead of just the biggos you're adding.

Two options:

* Specify which values you want to run the loop to limit it to just the new marketing spacers
* Ditch the fancy Sass loop all together and write it manually

Reply:
i'm gonna specify the new variables so the loop only goes through the new marketing spacers